src/widgets/speakers/ui/speakers.module.scss

```diff
@@ -0,0 +1,56 @@
+.speakers {
+  display: flex;
+  gap: 50px;
+  background-color: $color-gray-100;
+
+  .info {
+    width: 640px;
+    font-weight: $font-weight-medium;
+    color: $color-black;
+
+    .name {
+      margin-top: 16px;
+    }
+
+    .email-wrapper {
+      display: flex;
+
+      margin-top: 8px;
+
+      font-size: 18px;
+      font-style: normal;
+      line-height: 24px;
+
+      .email {
+        margin-left: 8px;
+      }
+
+      @include media-tablet {
+        font-size: 16px;
+        line-height: 20px;
+      }
+    }
+
+    @include media-laptop {
+      width: 100%;
+    }
+  }
+
+  .picture {
+    width: 294px;
+    height: 300px;
+    margin-left: auto;
```

Comment:
If I'm not mistaken Anya asked to avoid using margin if it can be changed. Here and below. `margin: 16px 0 0 0;`

Reply:
fixed